README.md

```diff
@@ -107,6 +107,50 @@ https://appacademy.io/alumni-engagement-agreement
 
 ## Workflow
 * So you want to contribute to cohort resources? you've come to the right spot
+### Jira Workflow 
+* [Jira Link](https://appacademyio.atlassian.net/secure/RapidBoard.jspa?rapidView=86&projectKey=IN&selectedIssue=IN-91)
+* This is a slightly modified outline of the workflow for the curriculumn team
```

Comment:
curriculum --> cohort resources

Reply:
I just went ahead and removed this sentence